### PR TITLE
fix imports that were causing thrown errors

### PR DESCRIPTION
**Pull request by ejfrancis**
After my previous fix for the import rule I ran it on a large codebase and found a few more issues that pop up. I'm not sure what's causing these to happen but adding safety checks fixed it and all tests are still passing.

### CODE REVIEW

**Comment by ejfrancis** [comment]
@AlexMost do you need anything else on this PR? I'd like to use this in a project of mine but these issues are preventing me atm

File: lib/ImportChecker.js [hunk 1]
```diff
@@ -27,6 +27,9 @@ class ImportChecker {
             return importPath === this.name;
         }
         if (this.regExp) {
+            if (typeof importPath !== 'string') {
```
Comment (ejfrancis) [inline lib/ImportChecker.js:30]: @AlexMost the null check for `requireArg` was merged in #31 but this issue still persists atm

**Comment by ejfrancis** [comment]
bump, this issue is preventing me from using the rule atm

**Comment by AlexMost** [comment]
Ah, sorry for the late reply, going to release new ver ASAP

**Comment by AlexMost** [comment]
you can try the new version `eslint-plugin-deprecate@0.8.4`. Thanks for your contribution!

**Pull request merged**